### ISSUE TITLE
Show List Size Calculation Error

### ISSUE THREAD

**Issue by jimmielightner** [SickRage/SickRage#3351]
SickRage Configuration Info:
Branch: master
Commit: 41d7d9d8f41c5cf4e73f85b2c3d417e4ff93bbb6
Database Version: 44.0
  Python Version: 2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:19:22) [MSC v.1500 32 bit (Intel)]
  SSL Version: OpenSSL 1.0.2h 3 May 2016
  OS: Windows-10-10.0.14393.693
  Locale: en_US.cp1252

**What you did**: When viewing the main Show List page, SickRage is reporting double the space used for a double-episode file (ex: S01E01E02). The total file size as reported by Windows is 5.19GB. SickRage Show List is reporting 10.39GB. 

**What happened**: SickRage reports double the actual storage for a single, double-episode (ex: S01E01E02) file on the main Show List page. This is different from the correct calculation within the individual show details.

**What you expected**: SickRage accurately reports file size information for a show from every view. When viewing the individual show details, the size is reported correctly.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by sharkykh** [comment]
This is a known issue. We fixed it twice, with two different solutions. (https://github.com/SickRage/SickRage/commit/2508a85d9426f46a6ca8c098a410e9c8a809062a, https://github.com/SickRage/SickRage/commit/9b27db90eb160210c1f8ef52f028f9a78151ef25)
However, the last time we did it (https://github.com/SickRage/SickRage/commit/9b27db90eb160210c1f8ef52f028f9a78151ef25), it caused issues for users running SickRage on Qnap,
and we weren't able to fix it yet.

**Comment by sharkykh** [comment]
@miigotu 
What if we added a show_size column in tv_shows table?
I mean, we would calculate tv_show every time an episode is added/removed,
or in every place we update a show in tv_shows (ideally we should only sum the sizes in one place) 

Would that be difficult? Do we have to have a new database version for this?
(If so I will open a milestone for the next database version to keep track of things - like flatten_folders)

**Comment by miigotu** [comment]
Yeah every db change needs a new DB version, so we need to make a big list of exactly what changes we need done to the DB and try to do quite a few at once.

show_size would speed up both home and displayShow
